code/game/objects/items/weapons/manuals.dm

```diff
@@ -1183,3 +1183,891 @@
+	desc = "A set of guidelines aiming at the safe conduct of all medical activities."
+	icon_state = "book7"
+	author = "Nanotrasen"
+	title = "Medical Standard Operating Procedures"
+	dat = {"
+
+		<html>
+				<head>
+				<style>
+				h1 {font-size: 15px; margin: 15px 0px 5px;}
+				li {margin: 2px 0px 2px 15px;}
+				ul {list-style: none; margin: 5px; padding: 0px;}
+				ol {margin: 5px; padding: 0px 15px;}
+				</style>
+				</head>
+				<body>
+				<font face="Verdana" color=black>
+
+				<h1><a name="Contents">Contents</a></h1>
+				<ol>
+					<li><a href="#foreword">Foreword</a></li>
+					<li><a href="#cmo">Chief Medical Officer</a></li>
+					<li><a href="#doc">Medical Doctor</a></li>
+					<li><a href="#chem">Chemist</a></li>
+					<li><a href="#gen">Geneticist</a></li>
+                    <li><a href="#viro">Virologist</a></li>
+                    <li><a href="#para">Paramedic</a></li>
+                    <li><a href="#psy">Psychologist</a></li>
+                    <li><a href="#surg">Surgery</a></li>
+                    <li><a href="#viral">Viral Outbreak Procedures</a></li>
+                    <li><a href="#cor">Coroner Procedures</a></li>
+                    <li><a href="#exo">Exotic Implants</a></li>
+				</ol>
+				<br><BR>
+
+				<h1><a name="foreword"><U><B>FOREWORD</B></U></a></h1><BR>
+				Job SOP should not be a considered a checklist of conditions to fire someone over, and should not be rigidly followed to the letter in detriment of circumstances and context.
+				As always, SOP can be malleable if the situation so requires, and the decision to punish a crewmember for breaching it ultimately falls onto the relevant Head of Staff,
+				for Department Members, or Captain, for the Head of Staff.<BR><BR>
+
+				<h1><a name="cmo"><B>Chief Medical Officer</B></a></h1><BR>
+				<ol>
+					<li>The Chief Medical Officer is permitted to carry a regular Defibrillator or a Compact Defibrillator on their person at all times;</li>
+                    <li>The Chief Medical Officer is permitted to carry a telescopic baton. In case Genetic Powers need to be forcefully removed, they are cleared to carry a Syringe Gun;</li>
+                    <li>The Chief Medical Officer is not permitted to allow the creation of poisonous or explosive mixtures in Chemistry without express consent from the Captain or, failing that, the presence of a clear and urgent danger to the integrity of the station, except of course in situations where Chemical Implants are required;</li>
+                    <li>The Chief Medical Officer is not permitted to allow the release of any virus without a full list of its symptoms, as well as the creation of a vial of antibodies, to be kept in a secure location. The virus may not have any harmful symptoms whatsoever, though neutral/harmless symptoms are permitted;</li>
+                    <li>The Chief Medical Officer must make sure that any cloneable corpses are, in fact, clone.</li>

```

Comment:
The Chief Medical Officer must make sure that any cloneable corpses are, in fact, **cloned**